examples/bowling_game_score_calculator/config.yml

```diff
@@ -0,0 +1,13 @@
+# Run the adam tests with unittest
```

Comment:
Comment needs to get updated?!

Reply:
Should this (bowling_game_score_calculator) be in here (this PR) after all?

Reply:
That comment is cruft; I've removed it. 

I *think* the `bowling_game_score_calculator` was part of the same documentation effort from the PyCon UK sprint. @trbedwards: was this intended to be in the same PR as `simple_math`? I may have messed up the rebasing or something.